purchase.adoc

```diff
@@ -153,9 +247,9 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
         "accuracyMeters": 65
       },
       "purchaseNumber": 9,
-      "userDisplayName": "Pär Svanström",
-      "userId": 5210,
-      "organizationId": 6927,
+      "userDisplayName": "Elna Westman",
```

Comment:
Cool. Yes, I think there's value in making it obvious that the name is made up (so outside parties won't believe we're leaking customer data).